images/release/golang-1.8/Dockerfile

```diff
@@ -31,6 +32,6 @@ RUN mkdir $TMPDIR && \
     git config --system user.name origin-release-container && \
     git config --system user.email none@nowhere.com
 
-WORKDIR /go/src/github.com/openshift/origin
+WORKDIR $WORKDIR
```

Comment:
Currently, BASETMPDIR is set to `"${TMPDIR:-/tmp}/openshift/${sub_dir}"`, which when using hack/env is currently being set by the release image as `/openshifttmp`.

Updating the images seemed like the path of least resistance without causing changes to users running the scripts locally that may not want their TMPDIR location overriden.